archinstall/lib/installer.py

```diff
@@ -130,13 +130,23 @@ def _verify_service_stop(self):
 		One such service is "reflector.service" which updates /etc/pacman.d/mirrorlist
 		We need to wait for it before we continue since we opted in to use a custom mirror/region.
 		"""
-		info('Waiting for time sync (systemd-timesyncd.service) to complete.')
 
-		while True:
-			time_val = SysCommand('timedatectl show --property=NTPSynchronized --value').decode()
-			if time_val and time_val.strip() == 'yes':
-				break
-			time.sleep(1)
+		if not storage['arguments'].get('skip_ntp', False):
+			info('Waiting for time sync (timedatectl show) to complete.')
```

Comment:
probably stupid question, why change  from `systemd-timesyncd.service` to` timedatectl show` ?